public/validators/main-albatross/NQ44 V95C ABVY RARR SBMC V8VE M6UH EJP0 RXYQ.json

```diff
@@ -1,8 +1,18 @@
+  "payoutSchedule": "After every epoch",
+  "contact": {
+    "email": "hi@nim.re",
+    "telegram": "@nim_re",
+    "twitter": "@nim_dot_re",
+    "discordInvitationUrl": "https://discord.gg/nimiq",
```

Comment:
The discord link is optional, using Nimiq's discord is not allowed.

Please remove it or provide your own server, thank you.